deployment/packer/cac.json

```diff
@@ -0,0 +1,94 @@
+{
+  "variables": {
+    "local_project_directory": ".",
+    "aws_access_key": "",
```

Comment:
Packer will the common AWS SDK paths (I believe) to grab keys, so passing them in might not be required. We might want to look at some of the authorization helpers in other projects for examples. That way this should be usable with MFA tokens as well, but would require running another command (to authenticate with MFA) before running the packer commands.